ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java

```diff
@@ -97,61 +96,61 @@ public final String getFullResourceName() {
   }
 
   public final String getScheduledExtn() {
-    String absoluteScheduleName = getJobParamValue( "ActionAdapterQuartzJob-StreamProvider-InputFile" );
+    // Jobs scheduled in PUC have a param with an input path on its own
+    String inputPath = getJobParamValue( "ActionAdapterQuartzJob-StreamProvider-InputFile" );
+
+    // Jobs scheduled in PDI only have a single field with combined input/output paths
+    // in this format "input file = /home/admin/myTransformation.ktr:outputFile = /home/admin/myTransformation.*"
+    String inputOutputPaths = getJobParamValue("ActionAdapterQuartzJob-StreamProvider");
+
+    String fileExtension = "";
+    if ( StringUtils.isNotBlank( inputPath ) ) {
+      fileExtension = inputPath.substring( inputPath.lastIndexOf( '.' ) + 1 );
+    } else if ( StringUtils.isNotBlank( inputOutputPaths ) ) {
+      String splitInputFilePath = inputOutputPaths.substring(0, inputOutputPaths.indexOf(":outputFile = "));
```

Comment:
If we're trying to be totally bulletproof here, that's actually a bit of a problem because `:outputFile =` (without the trailing space) is an allowed name for a directory. With the trailing space is illegal though.